package.json

```diff
@@ -8,7 +8,7 @@
     "coverage:builder": "istanbul cover --report=html --dir=./coverage/builder ./node_modules/jasmine/bin/jasmine.js JASMINE_CONFIG_PATH=jasmine.json",
     "coverage:runtime": "node ./utils/cli-test.js dev-runtime",
     "coverage:src-app": "node ./utils/cli-test.js dev-src-app",
-    "e2e": "webdriver-manager update --standalone false --gecko false && npm run protractor",
+    "e2e": "node ./utils/cli-chromedriver-manager-update && npm run protractor",
```

Comment:
Calling `webdriver-manager` directly will install the latest version of ChromeDriver.  Unfortunately, Appveyor is not currently in sync with that.

This behavior is exactly why we created the `chromedriver-version-matcher` so that we can match the version of ChromeDriver to the version of Chrome.  

Solving this meant refactoring the code out of `cli/e2e.js` in to `cli/utils/chromedriver-manager.js` and `utils/cli-chromedriver-manager-update.js`